packages/webapp/pages/squads/[handle]/moderate.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { ReactElement } from 'react';
+import { ManageSquadPageContainer } from '@dailydotdev/shared/src/components/squads/utils';
+import {
+  SquadTab,
+  SquadTabs,
+} from '@dailydotdev/shared/src/components/squads/SquadTabs';
+import { SquadModerationList } from '@dailydotdev/shared/src/components/squads/moderation/SquadModerationList';
+import {
+  PageHeader,
+  PageHeaderTitle,
+} from '@dailydotdev/shared/src/components/layout/common';
+import {
+  Button,
+  ButtonVariant,
+} from '@dailydotdev/shared/src/components/buttons/Button';
+import { ArrowIcon } from '@dailydotdev/shared/src/components/icons';
+import { GetServerSidePropsContext, GetServerSidePropsResult } from 'next';
+import { ParsedUrlQuery } from 'querystring';
+import { oneHour } from '@dailydotdev/shared/src/lib/dateFormat';
+import {
+  getSquadStaticFields,
+  SquadStaticData,
+  verifyPermission,
+} from '@dailydotdev/shared/src/graphql/squads';
+import { useSquad } from '@dailydotdev/shared/src/hooks';
+import { SourcePermissions } from '@dailydotdev/shared/src/graphql/sources';
+import { getLayout as getMainLayout } from '../../../components/layouts/MainLayout';
+
+interface ModerateSquadPageProps {
+  squad: SquadStaticData;
+}
+
+export default function ModerateSquadPage({
+  squad: squadProps,
+}: ModerateSquadPageProps): ReactElement {
+  const { squad, isLoading } = useSquad({ handle: squadProps.handle });
+  const isModerator = verifyPermission(squad, SourcePermissions.ModeratePost);
+
+  if (isLoading || !squad) {
+    return null;
+  }
+
+  return (
+    <ManageSquadPageContainer>
+      <PageHeader className="border-b-0">
+        <Button
+          href={`/squads/${squad.handle}`}
+          icon={<ArrowIcon className="-rotate-90" />}
+          tag="a"
+          variant={ButtonVariant.Tertiary}
+        />
+        <PageHeaderTitle className="typo-title3">Squad</PageHeaderTitle>
+      </PageHeader>
+      <SquadTabs
+        active={SquadTab.PendingPosts}
+        handle={squadProps.handle}
+        pendingCount={squad?.moderationPostCount}
+        showSettings={isModerator}
+      />
+      <SquadModerationList squad={squad} />
+    </ManageSquadPageContainer>
+  );
+}
+
+interface SquadPageParams extends ParsedUrlQuery {
+  handle: string;
+}
+
+export async function getServerSideProps({
+  params,
+  res,
+}: GetServerSidePropsContext<SquadPageParams>): Promise<
+  GetServerSidePropsResult<ModerateSquadPageProps>
+> {
+  const setCacheHeader = () => {
```

Comment:
This might cause an issue if you change the setting of the squad?
Not sure but check here:
https://dailydotdev.slack.com/archives/C07GFU7DX8X/p1731565434782739

Reply:
Right. We should maybe leave this one out since it won't really matter to have this cached, or at least lower the time.

Reply:
I think that, most of the time, users will have already fetched the squad client-side by the time they land on the /moderate page, and the squad query is usually already freshly cached.

In my opinion, we could try making this page client-side only and remove this fetch since we mainly want fresh data. What do you think?